lib/chess/core/chess.rb

```diff
@@ -31,7 +31,11 @@ module Chess
   EXIT_COMMAD = :exit_game
   COMMAND_SUCCES = :ok
 
-  CASTLING_FAILS = :cant_castle
+  CASTLING_FAILS = :cant_castling
```

Comment:
this constant name is already repeated on line 14

Reply:
yeah, I'm correcting this in other PR